src/clims/services/base_extensible_service.py

```diff
@@ -61,24 +62,24 @@ def all(self):
         # which in turn would return the wrapper when materialized. For that to work smoothly,
         # we'll need to look into implementation details of django querysets.
 
-        # TODO: how does the prefetch perform when fetching all objects like this
-        for entry in self._all_qs():
-            yield self.to_wrapper(entry)
+        # TODO: The `to_wrapper` method is NOT on this mixin, which makes this difficult to reason
```

Comment:
I agree with this. Do you think we should fix it in this PR, or do it later?

Reply:
I think I would like to leave it for later, as I has a ton of PRs to get in :|